shared/InitialConfig.ts

```diff
@@ -0,0 +1,20 @@
+// Initial configs used by app if network is not available when app is opened for first time. once network is available response is cached and used from then
+// Note: Need to keep this config in sync with actual mimoto response.
+
+export const INITIAL_CONFIG = {
+  allProperties: {
+    modelDownloadMaxRetry: '10',
+    audience: 'ida-binding',
+    allowedInternalAuthType: 'otp,bio-Finger,bio-Iris,bio-Face',
+    vcDownloadMaxRetry: '10',
+    minStorageRequiredForAuditEntry: '2',
+    minStorageRequired: '2',
+    vcDownloadPoolInterval: '6000',
+    issuer: 'residentapp',
+    allowedAuthType: 'demo,otp,bio-Finger,bio-Iris,bio-Face',
+    allowedEkycAuthType: 'demo,otp,bio-Finger,bio-Iris,bio-Face',
+    warningDomainName: 'https://api.qa-inji.mosip.net',
```

Comment:
Remove warningDomainName and faceSdkModelUrl from caching as it depends on api response.
Let app handles these errors if these values are missing

Reply:
removed

Reply:
removed now. after removing also app works fine